### ISSUE TITLE
timestamps aren't pulled from comment replies

### ISSUE THREAD

**Issue by nadomodan** [HolodexNet/Holodex#655]
Some timestampers split timestamps into multiple comments that are replies to their first comment, holodex only shows timestamps on timeline from the root comment but not from replies to that comment

**Comment by sphinxrave** [comment]
Yeah it's kind of a known issue at the moment. We're experimenting with modifying the comment crawler, but with youtube being what it is, we can only pull top (?first?) 5 comments at the time for each comment thread - a lot of extra data but not necessarily getting what we needed..

An alternate strategy is being tested on, but requires that you go to our discord and manually ping the video to be updated.

![image](https://user-images.githubusercontent.com/62570796/213721989-b849d782-6103-4e42-beab-07b9689e453a.png)

Let us know if you think this is an acceptable solution @nadomodan , but otherwise we'll have to dedicate a separate crawler to figuring out which threads deserve a deeper look.
